eth/protocols/snap/sync.go

```diff
@@ -2756,3 +2846,23 @@ func estimateRemainingSlots(hashes int, last common.Hash) (uint64, error) {
 	}
 	return space.Uint64() - uint64(hashes), nil
 }
+
+// capacitySort implements the Sort interface, allowing sorting py peer message
+// throughput (highest first).
+type capacitySort struct {
+	ids  []string
+	caps []float64
+}
+
+func (s *capacitySort) Len() int {
+	return len(s.ids)
+}
+
+func (s *capacitySort) Less(i, j int) bool {
+	return s.caps[i] > s.caps[j]
+}
```

Comment:
The method is `Less`, but you implemented the opposite. Isn't it more correct to implement `Less` (as in "is Capacity i less than j", and then use `sort.Reverse` if you want in some other order?

Reply:
Sure